src/SockPerf.cpp

```diff
@@ -2928,15 +2922,13 @@ static int set_sockets_from_feedfile(const char *feedfile_name)
 			sock_type = SOCK_DGRAM;
 			ip = strtok(line, ":");
 		}
-#ifdef  USING_VMA_EXTRA_API
 		if (sock_type == SOCK_DGRAM && s_user_params.mode == MODE_CLIENT){
-			if (s_user_params.fd_handler_type == SOCKETXTREME && !s_user_params.client_bind_info.sin_port && !s_user_params.client_bind_info.sin_addr.s_addr) {
-				log_msg("socketxtreme requires forcing the client side to bind to a specific ip address (client_ip) option");
+			if (s_user_params.fd_handler_type == SOCKETXTREME && s_user_params.client_bind_info.sin_addr.s_addr == INADDR_ANY) {
+				log_msg("socketxtreme requires forcing the client side to bind to a specific ip address (use --client_ip) option");
```

Comment:
Do we need to check for the port ?

Reply:
To use socketXtreme mode we should specify specific interface. To do this in current implementation `--client_ip` option is used. Port is useless.

Reply:
:+1: